src/tasks/tap/path.js

```diff
@@ -0,0 +1,55 @@
+const { getTapPath, getTapConfig, getTapPackage } = require('KegRepos/cli-utils')
+const { generalError } = require('KegUtils/error')
+
+/**
+ * Prints out a tap's root, config, or package.json file path.
+ * @param {Object} args - arguments passed from the runTask method
+ */
+const printPath = args => {
+  const { params } = args
+  const { tap, config, package } = params
+
+  !tap && generalError('Cannot print config without a tap parameter.')
+
+  const tapPath = getTapPath(args.globalConfig, tap)
+  const [ , configPath ] = getTapConfig({ name: tap })
+  const [ , packagePath ] = getTapPackage({ name: tap })
+  
+  if (package) return console.log(packagePath)
```

Comment:
Why use `console.log` here, and not `Logger.stdout` ?
* `console.log` always adds a `\n` at the end where calling `stdout` does not

Reply:
I remember the `keg tap print` task I could not get `open $(keg evf print package.repository.url)` unless I used `Logger.stdout` but that doesn't seem to be needed here.

Reply:
Interesting, works for me